docs/app/pages/components/components-sections/tables/filters/filters.component.ts

```diff
@@ -158,19 +164,28 @@ export class ComponentsFiltersComponent extends BaseDocumentationSection impleme
         // apply a newly added filter
         if (event instanceof FilterAddEvent) {
             this.applyFilter(event.filter);
+
+            // announce the selection
+            this._announcer.announce(`Filter ${event.filter.name} selected.`);
```

Comment:
Probably should rename the function containing this, since we now have an event emitter with almost the same name. My suggestion would be to use the `(events)` emitter for LiveAnnouncer and `(filterChange)` to update the table's filtered content. Then if one modifies the `filter` array directly, the table state would still be updated.